### ISSUE TITLE
Google App Engine - ImportError: No module named grp 

### ISSUE THREAD

**Issue by zippy84** [cherrypy/cheroot#138]
I'm trying to run CherryPy in Googles App Engine. Because I only can use Python 2.7 I installed an older version of CherryPy with pip install -t /home/zippy/ecl/ecl_cherrypy/lib "CherryPy==17.4.0". 

And now I get:

```
Traceback (most recent call last):
  File "/home/zippy/ecl/google_appengine/google/appengine/runtime/wsgi.py", line 240, in Handle
    handler = _config_handle.add_wsgi_middleware(self._LoadHandler())
  File "/home/zippy/ecl/google_appengine/google/appengine/runtime/wsgi.py", line 299, in _LoadHandler
    handler, path, err = LoadObject(self._handler)
  File "/home/zippy/ecl/google_appengine/google/appengine/runtime/wsgi.py", line 85, in LoadObject
    obj = __import__(path[0])
  File "/home/zippy/ecl/ecl_cherrypy/ecl.py", line 9, in <module>
    import cherrypy
  File "/home/zippy/ecl/ecl_cherrypy/lib/cherrypy/__init__.py", line 76, in <module>
    from . import _cprequest, _cpserver, _cptree, _cplogging, _cpconfig
  File "/home/zippy/ecl/ecl_cherrypy/lib/cherrypy/_cprequest.py", line 13, in <module>
    from cherrypy import _cpreqbody
  File "/home/zippy/ecl/ecl_cherrypy/lib/cherrypy/_cpreqbody.py", line 135, in <module>
    import cheroot.server
  File "/home/zippy/ecl/ecl_cherrypy/lib/cheroot/server.py", line 102, in <module>
    import grp
  File "/home/zippy/ecl/google_appengine/google/appengine/tools/devappserver2/python/runtime/sandbox.py", line 1095, in load_module
    raise ImportError('No module named %s' % fullname)
ImportError: No module named grp
```
Could you say what's wrong with it?

`python2 -c "import grp"` works fine.

**Comment by zippy84** [comment]
> I think, as a workaround, pinning Cheroot<6.2.0 should temporarily fix the issue for you until it's fixed in upstream.

You are right.

`pip install -t /home/zippy/ecl/ecl_cherrypy/lib "CherryPy==11.0.0"` is not a solution, because the problem is located in Cheroot.

I already find a solution by downgrading Cheroot to 5.8.3.

**Comment by webknjaz** [comment]
@zippy84 mind submitting a PR to improve this situation?

**Comment by zippy84** [comment]
I can do that, but how should we handle this `ImportError`? I found out, that you are using grp only for informative purpose, to retrieve the name of the group ID. I can wrap `import grp` into  an `try ... except ...` and raise an `RuntimeError` after the `if ... else ...` starting at https://github.com/cherrypy/cheroot/blob/8aca957efef4377f150245d7e63f4453d0de16ab/cheroot/server.py#L1374.

What do you think?

**Comment by zippy84** [comment]
Let's find out.

**Comment by webknjaz** [comment]
I'd prefer `try:`/`except ImportError:`, because such way is more generic.

**Comment by zippy84** [comment]
Generic, but a successful import takes too much time.

**Comment by webknjaz** [comment]
Well, iterating over packages may be unpredictably longer because of the varying number of modules.

**Comment by webknjaz** [comment]
Resolved by #140
